packages/providers/fal/image-to-video/fal_image_to_video/models/base.py

```
@@ -7,7 +7,8 @@
 from typing import Dict, Any, Optional
 import fal_client
 
-from ..config.constants import MODEL_ENDPOINTS, MODEL_DISPLAY_NAMES, MODEL_PRICING
+from ai_content_pipeline.registry import ModelRegistry
+import ai_content_pipeline.registry_data  # noqa: F401
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)